.github/workflows/test_weaver-fabric-fabric-asset-transfer.yaml

```diff
@@ -0,0 +1,367 @@
+# Copyright IBM Corp. All Rights Reserved.
```

Comment:
If you want to use this file for `satp_test`, may be call it `test_weaver-satp.yaml`

Reply:
In this file I thought it is better to have the test done between two fabric networks only. So maybe test_weaver-fabric-fabric-satp.yaml